source/unified-test-format/unified-test-format.md

```diff
@@ -582,6 +582,8 @@ The structure of this object is as follows:
             See the [Client-Side Encryption test README](../client-side-encryption/tests/README.md#credentials) for
             instructions to obtain test credentials.
 
+        - `keyExpirationMS`: Optional integer. The cache expiration time for data encryption keys.
```

Comment:
Add new option to the [User Facing API](https://github.com/mongodb/specifications/blob/11022ca4f21377f00827918ebbdf5ea3dbdb7d4a/source/client-side-encryption/client-side-encryption.md#user-facing-api) sections of the spec.